app/helpers/storytime/application_helper.rb

```diff
@@ -27,6 +27,14 @@ def active_nav_item_class(controller, type = nil)
       'class="active"'.html_safe if controller == current_controller
     end
 
+    def active_blog_item_class(blog)
+      path_arr = request.path.split("/")
+
+      return unless path_arr.include?("blogs")
```

Comment:
:+1: @bcroesch , In the result it can be 
```ruby
def active_blog_item_class(blog)
  'class="active"'.html_safe if params[:blog_id] == blog.slug && params[:controller] == "storytime/dashboard/posts"
end
```
or 

```ruby
def active_blog_item_class(blog)
  'class="active"'.html_safe if params[:blog_id] == blog.slug && controller_name == "posts"
end
```